subgraphs/compound-forks/dforce/README.md

```diff
@@ -0,0 +1,110 @@
+
+### Supply-Side Revenue USD
+
+Portion of the Total Revenue allocated to the Supply-Side. The total interest (revenue) is split between depositors (supply-side) and the protocol based on the value of the `reserveRatio`.
+
+Sum across all Markets:`Total Revenue USD - Protocol-Side Revenue USD`
+
+### Total Unique Users
+
+Count of Unique Addresses which have interacted with the protocol via any transaction
+
+`Deposits`
+
+`Withdrawals`
+
+`Borrows`
+
+`Repays`
+
+`Liquidations`
+
+### Reward Token Emissions Amount
+
+The dForce lending RewardDistributor contract computes the rewards emissions to lenders and borrowers when users interacts with the contracts, and emit `RewardDistributed` event when emissions are distributed to user account. Since there is no event for market-level emissions, the daily emissions for a market have to be computed using the `distributionSpeed` storage in the RewardDistributor contract multiplied with `BLOCKS_PER_DAY`:
+
+Reward emissions for lenders: `distributionSupplySpeed * priceOracle.getUnderlyingPrice(iToken) * BLOCKS_PER_DAY`
+
+Reward emissions for borrowers: `distributionSpeed * priceOracle.getUnderlyingPrice(iToken) * BLOCKS_PER_DAY`
+
+### Protocol Controlled Value
+
+Not applicable.
+
+## Links
+
+- Protocol: https://dforce.network/
+- Analytics: [Not found]
+- Docs: https://docs.dforce.network/
+- Smart contracts: https://github.com/dforce-network/LendingContractsV2
+- Deployed addresses: https://developers.dforce.network/lend/lend-and-synth/deployed-contracts
+- Official subgraph: [Not found]
+
+## Build
+
+- Initialize subgraph (Subgraph Studio):
+  ```
+  graph init --product subgraph-studio
+  --from-contract <CONTRACT_ADDRESS> [--network <ETHEREUM_NETWORK>] [--abi <FILE>] <SUBGRAPH_SLUG> [<DIRECTORY>]
+  ```
+- Initialize subgraph (Hosted Service):
+  ```
+  graph init --product hosted-service --from-contract <CONTRACT_ADDRESS> <GITHUB_USER>/<SUBGRAPH_NAME>[<DIRECTORY>]
+  ```
+- Generate code from manifest and schema: `graph codegen`
+- Build subgraph: `graph build`
+
+## Deploy
+
+- Authenticate (just once): `graph auth --product hosted-service <ACCESS_TOKEN>`
+- Deploy to Subgraph Studio: `graph deploy --studio <SUBGRAPH_NAME>`
+- Deploy to Hosted Service: `graph deploy --product hosted-service <GITHUB_USER>/<SUBGRAPH_NAME>`
+
+## Hosted Subgraph
+
+- Hosted subgraph: https://thegraph.com/hosted-service/subgraph/tnkrxyz/dforce (mainnet)
+- Hosted subgraph: https://thegraph.com/hosted-service/subgraph/tnkrxyz/dforce-bsc (BSC)
+- Hosted subgraph: https://thegraph.com/hosted-service/subgraph/tnkrxyz/dforce-optimism (Optimism)
+- Hosted subgraph: https://thegraph.com/hosted-service/subgraph/tnkrxyz/dforce-arbitrum (Arbitrum-One, empty subgraph)
+
+## Smart Contracts Interactions
+
+![dforce-lending](https://lh6.googleusercontent.com/ZEc8wddU1Etx-ay_fwnat2dgje8yGcOd8EK1vo5JSxJp3GM7qK5bD-bCvf3zxv4m1j5ClAfoCKiPzWNcqjS_XRjrThyz7vGA-ACalWb2XmwqEwGcLMQLSxVtySU7NPGjxPNOqqoB "dforce-lending")
```

Comment:
you should put the image under `docs` in the root directory

Reply:
fixed.